### ISSUE TITLE
SAIGE_fitGLMM_fast.o Error 1 during installation

### ISSUE THREAD

**Issue by kvedes** [weizhouUMICH/SAIGE#258]
Hi

I'm running a machine with Ubuntu 18.04 having issues installing SAIGE.

I installed the conda environment as described in step 1 and 2 in the README. For step 3 I installed MetaSKAT with the following code, since it cannot be downloaded from CRAN anymore:

```
wget https://cran.r-project.org/src/contrib/Archive/MetaSKAT/MetaSKAT_0.80.tar.gz
sudo Rscript -e 'install.packages("MetaSKAT_0.80.tar.gz", repos=NULL, type="source")'
```

In step 4 I run the command:

`devtools::install_github("weizhouUMICH/SAIGE")`

I get the error:

```
make: *** [/home/ubuntu/miniconda3/envs/RSAIGE/lib/R/etc/Makeconf:175: SAIGE_fitGLMM_fast.o] Error 1
ERROR: compilation failed for package ‘SAIGE’
* removing ‘/home/ubuntu/miniconda3/envs/RSAIGE/lib/R/library/SAIGE’
Error: Failed to install 'SAIGE' from GitHub:
  (converted from warning) installation of package ‘/tmp/RtmpBwTfYs/file28d861537a37/SAIGE_0.42.1.tar.gz’ had non-zero exit status
```

Any suggestions what might cause this?

**Comment by weizhou0** [comment]
Hi @kvedes, 

Can you please sharing more details for the error message? You may output the screen output to a file using 2> err.txt

Thanks,
Wei

**Comment by kvedes** [comment]
Hi @weizhou0 

Thanks for looking into this. Below you can see the full error.

Let me know if you need more information.

Thanks

[err.txt](https://github.com/weizhouUMICH/SAIGE/files/5446516/err.txt)

**Comment by cgpu** [comment]
I have the exact same error, subscribing for any information that would help.

**Comment by henwin** [comment]
What version of RcppArmadillo do you have installed?  When I tried building SAIGE using the latest CRAN version (0.10.1.0.0), I got the exact same error as you.  Reverting to RcppArmadillo 0.9.900.3.0 allowed me to build SAIGE without problems under R-4.0.3.

**Comment by weizhouUMICH** [comment]
We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
